### PR TITLE
Bug 957224 - Change sound notification duration r=etienne

### PR DESCRIPTION
**Pull request by benoittgt**
Patch was made for the previous version : https://bugzilla.mozilla.org/show_bug.cgi?id=957224
Add some tests to verify the 4000ms duration.

### CODE REVIEW

File: apps/system/test/unit/notifications_test.js [hunk 1]
```diff
@@ -237,6 +237,18 @@ suite('system/NotificationScreen >', function() {
       assert.ok(playSpy.calledOnce);
     });
 
+    test('sound notifications lasts 4 seconds', function() {
```
Comment (etiennesegonzac) [inline apps/system/test/unit/notifications_test.js]: -> `test('it should stop playing after 4 seconds', function() {`

**Comment by try-server-hook** [comment]
[benoittgt Benoit (benoittgt)](https://github.com/benoittgt) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cc0d191c008b)

**Comment by try-server-hook** [comment]
[benoittgt Benoit (benoittgt)](https://github.com/benoittgt) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5ca46a41f590)

**Comment by try-server-hook** [comment]
[benoittgt Benoit (benoittgt)](https://github.com/benoittgt) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c47aaa91b3ba)

**Comment by try-server-hook** [comment]
[benoittgt Benoit (benoittgt)](https://github.com/benoittgt) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6e7e76b08c21)

**Comment by try-server-hook** [comment]
[benoittgt Benoit (benoittgt)](https://github.com/benoittgt) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=22086406ebbe)

**Pull request merged**